pages/download.vue

```diff
@@ -10,16 +10,28 @@ const links = await api.downloadLinksGet();
 
 function isAppleSilicon() {
   try {
-    const canvas = document.createElement("canvas");
-    const gl = canvas.getContext("webgl");
+    // Best guess if the device uses Apple Silicon: https://stackoverflow.com/a/65412357
```

Comment:
I'd rather like to use an extension here. The benefit is that other people will quickly get edge cases and improve it for us. The here linked stackoverflow question has an answer with a link to this package: https://github.com/faisalman/ua-parser-js

Reply:
I agree with you in principle and we're actually using ua-parser-js for differentiating the OS. But for detecting Apple Silicon ua-parser-js relies on a new browser feature that only works on Chrome. I created an issue with them: https://github.com/faisalman/ua-parser-js/issues/732